cognite/client/_api/time_series.py

```diff
@@ -586,13 +641,19 @@ def upsert(
                 >>> new_time_series = TimeSeries(external_id="new_timeSeries", description="New timeSeries")
                 >>> res = c.time_series.upsert([existing_time_series, new_time_series], mode="replace")
         """
+        api_subversion = self._get_subapiversion_update(item)
+        if mode == "replace":
+            api_subversion = "beta"
+            self._unit_warning.warn()
```

Comment:
It will be awkward to implement, but I think `replace` without using the new features should not wipe units.

Maybe this could be an "easy" solution; adding something like `in_beta` to `PropertySpec`:

```py
@dataclass
class PropertySpec:
    name: str
    is_container: bool = False
    is_nullable: bool = True
    in_beta: bool = False
```

Reply:
Fixed.